tutorials/MultiQubitSystems/MultiQubitSystems.ipynb

```diff
@@ -345,6 +345,7 @@
    },
    "outputs": [],
    "source": [
+    "%config dump.basisStateLabelingConvention=\"Bitstring\"\n",
```

Comment:
We could add a note (before or after this cell) saying that you can change the format used by the DumpMachine to bitstring, little endian or big endian using `%config` - this way the default will be bitstring but we'll teach the learner the tools to switch